core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -2147,15 +2147,17 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
           TestUtils.waitUntilTrue(() => {
             val matching = client.listConsumerGroups.all.get.asScala.filter(group =>
               group.groupId == testGroupId &&
-                group.state.get == ConsumerGroupState.STABLE)
+                group.state.get == ConsumerGroupState.STABLE &&
+                group.groupState.get == GroupState.STABLE)
```

Comment:
Just a query: Why do we need to check `group.state.get` && `group.groupState.get` but just not `group.groupState.get` as the other method is deprecated and uses groupState itself internally?

Reply:
I want to ensure that they give the same result and continue to give the same result in the future. I know it should be obvious that they do, but a lot of testing is like that. In 5.0, we'll remove `ConsumerGroupState` entirely.

Reply:
Thanks for explaining.